react/src/components/Weather/TodaysWeather/TodaysWeather.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component, useEffect, useState } from "react";
+import Weather from './TodaysWeather.module.css';
+import CurrentWeather from '../CurrentWeather/CurrentWeather';
+import { Dimmer, Loader } from 'semantic-ui-react';
+
+export default function Pogoda_na_dzis() {
```

Comment:
```suggestion
const todaysWeather = () => {
```

+ na dole trzeba dodać `export default todaysWeather`, ponieważ `export default` nie działa z funkcją strzałkową w tej samej linii:

`export default const todaysWeather` - źle